Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:18-slim
+
+WORKDIR /app
+
+COPY . .
```

Comment:
Please only add the bundled application here

Reply:
I propose we build the bundled application in the container instead of the using the artifact from the build pipeline since otherwise the Dockerfile will be dependent on external builds (the build pipeline) and users will have a harder time trying to build the image themselves.